templates/java/org/yarp/Loader.java.erb

```diff
@@ -15,38 +15,59 @@ public class Loader {
         return new Loader(serialized, source).load();
     }
 
+    // Overridable methods
+
+    public Charset getEncodingCharset(String encodingName) {
+        encodingName = encodingName.toLowerCase(Locale.ROOT);
+        if (encodingName.equals("ascii-8bit")) {
+            return StandardCharsets.US_ASCII;
+        }
+        return Charset.forName(encodingName);
+    }
+
+    public <%= JAVA_STRING_TYPE %> bytesToString(byte[] bytes) {
```

Comment:
Maybe it should be `bytesToIdentifier()` or so and `JAVA_IDENTIFIER_TYPE` but that's so long.

Reply:
I renamtes to `bytesToName()`.
Maybe `JAVA_NAME_TYPE` would be good for the constant?